README.md

```diff
@@ -1,17 +1,47 @@
-
-We tried to focus on the developper experience, meaning that working with it should be as simple as possible.
+<h1 align="center">
+  Mozaic Design System
+</h1>
+
+> Mozaic is an open-source design system built by Adeo. A design system to reinforce team’s delivery time, 
+> improve quality of deliverables and thus help set up a coherent customer experience.
+
+<p align="center">
+  <a href="https://github.com/carbon-design-system/carbon/blob/master/LICENSE">
+    <img src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)" alt="Prettier badge" />
+  </a>
+  <a href="https://mozaic.adeo.cloud/">
+    <img src="https://img.shields.io/badge/smallchat-talk%20with%20us-green?style=flat-square)" alt="small chat" />
+  </a>
+  <a href="https://lerna.js.org/">
+    <img src="https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg" alt="Maintained with Lerna" />
+  </a>
+  <a href="https://mozaic.adeo.cloud/Contributing/Developers/InstallForDev/">
+    <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome" />
+  </a>
+</p>
+
+## Getting started
+If you are just getting started, check out our documentation: https://mozaic.adeo.cloud/
+
+If your are looking for JS components, we have community-contributed components for the following technologies: 
+- [Freemarker](https://github.com/adeo/mozaic-freemarker)
+- [Vue](https://github.com/adeo/mozaic-vue)
+- [Angular](https://github.com/adeo/mozaic-angular)
+- [React](https://github.com/adeo/mozaic-react)
+
+If you're trying to find something specific, here's a list of our main packages that we support!
+
+| Package name                                  | Description                                                                                                                                                                             |
+| --------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`@mozaic-ds/styles`](./packages/styles)                 | All the CSS and SCSS files that constitute the Mozaic SCSS framework.                                                                                                                                                 |
+| [`@mozaic-ds/tokens`](./packages/tokens)                 | Tokens are theming/styling constants defined in a JSON file. They are trasnpiled into multiple format to be shared across plateforms (SCSS, IOS, ANDROID...).                                                     |
+| [`@mozaic-ds/css-dev-tools`](./packages/css-dev-tools)     | Right now the tools are composed only of SCSS/PostCSS tools.                                                                                                                   |
```

Comment:
~~Right now the tools are composed only of SCSS/PostCSS tools.~~
=> Tools for css like PostCSS plugins and linters config